File: java/lib/src/main/java/io/openweb3/xwebhook/EndpointListOptions.java
package io.openweb3.xwebhook;

import io.openweb3.xwebhook.models.Ordering;

public class EndpointListOptions extends ListOptions {
    private Ordering order;

    public void setOrder(final Ordering order) {
        this.order = order;
    }

    public Ordering getOrder() {
        return this.order;
    }
}

File: kotlin/lib/src/main/kotlin/EndpointListOptions.kt
package io.openweb3.xwebhook.kotlin

import io.openweb3.xwebhook.kotlin.models.Ordering

class EndpointListOptions : ListOptions() {
    var order: Ordering? = null

    fun order(order: Ordering) = apply { this.order = order }

    override fun iterator(iterator: String) = apply { super.iterator(iterator) }

    override fun limit(limit: Int) = apply { super.limit(limit) }
}
